### ISSUE TITLE
Exit on checking for a valid network configuration

### ISSUE THREAD

**Issue by SuperDOS** [FriendsOfMDT/PSD#99]
Can't figure out what's wrong, booting from USB and it starts the checks but exit on  checking for a valid network configuration and a cmd prompt is shown.

`PSDStart.ps1: Invoking DHCP refresh...	PSDStart.ps1:447	2023-10-20 17:08:56	1744 (0x06D0)
PSDStart.ps1: We have at least one network adapter with an IP address, continuing...	PSDStart.ps1:471	2023-10-20 17:09:06	1744 (0x06D0)
PSDStart.ps1: Looking for PSDeployRoots in the usual places...	PSDStart.ps1:490	2023-10-20 17:09:06	1744 (0x06D0)
PSDStart.ps1: Property PSDeployRoots exists.	PSDStart.ps1:492	2023-10-20 17:09:06	1744 (0x06D0)
PSDStart.ps1: Testing PSDDeployRoots value: https://mdt01.domain.local/psdeploy	PSDStart.ps1:495	2023-10-20 17:09:06	1744 (0x06D0)
`

I can open a notepad window and browse \\mdt01.domain.local\psdeploy without problems with the credentials in bootstrap.ini 
`[Default]
UserDomain=domain.local
UserID=svc-mdt01
UserPassword=password@1`

**Comment by pimpmyname2** [comment]
1. Are you able to directory listening if you open up the browser and navigate to "https://mdt01.domain.local/psdeploy"?

2. Make sure your UserDomain is pingable, try use nslookup PSDSERVERIP and if UserDomain isnt the same as the result you got from nslookup, thats most likely the cause if its not the thing above.

**Comment by SuperDOS** [comment]
Yes I can browse https://mdt01.domain.local/psdeploy and \\mdt01.domain.local\psdeploy also the userdomain is pingable.
PSDSERVERIP is resolved, userdomain is the active directory domain no the local PSD-server, I will try with a local account instead.

**Comment by SuperDOS** [comment]
Yes a local account on the mdt01 works, is this a bug since we've used domain account in the regular mdt deployment?

**Comment by pimpmyname3** [comment]
I would suggest checking folder permissions for the specified domain account. Ensure that the Digest Authentication is enabled in your IIS PSDProduction folder. Additionally, try to attempt a manual login with x specified domain account on the website to ensure its functionality

**Comment by SuperDOS** [comment]
Thanks it might been an issue with authentication on the network port so I will close this.
